components/section/info/src/index.js

```diff
@@ -4,12 +4,16 @@ import PropTypes from 'prop-types'
 const baseClass = 'sui-SectionInfo'
 
 class SectionInfo extends Component {
+  displayTitle (title) {
```

Comment:
This function is returning a boolean. The name should be *isTitleDisplayed* or *shouldDisplayTitle* or something like that.